src/formatDate.js

```diff
@@ -50,7 +50,44 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat[fromFormat.length - 1];
+  const toSeparator = toFormat[toFormat.length - 1];
+  const arrDate = date.split(fromSeparator);
+  const result = [];
+
+  if (toFormat.join('').length !== fromFormat.join('').length) {
+    for (let i = 0; i < 3; i++) {
+      const value = fromFormat[i];
+
+      switch (true) {
+        case value === 'YY' && +arrDate[i] >= 30:
+          arrDate.unshift('19' + arrDate[i]);
+          fromFormat.unshift('YYYY');
+          break;
+
+        case value === 'YY' && +arrDate[i] < 30:
+          arrDate.unshift('20' + arrDate[i]);
+          fromFormat.unshift('YYYY');
+          break;
+
+        case value.length === 4:
+          fromFormat.unshift('YY');
+          arrDate.unshift(arrDate[i].slice(2));
+          break;
+      }
+    }
```

Comment:
let's organize your code 
```suggestion
 switch (value) {
  case 'YY':
    +arrDate[i] => 30 
      ? arrDate.unshift('19' + arrDate[i])
      : arrDate.unshift('20' + arrDate[i]);
    fromFormat.unshift('YYYY');
    break;

  case 'YYYY':
    fromFormat.unshift('YY');
    arrDate.unshift(arrDate[i].slice(2));
    break;

  default:
    break;
}
```

Reply:
I changed code with ternary operator, but it doesn`t  work. In my code I replaced the arrow "+arrDate[i] => 30" with greater than or equal